content/posts/webpack-code-splitting-separando-seus-modulos-para-uma-performance-incrivel.md

```diff
@@ -0,0 +1,470 @@
+---
+title: 'webpack Code Splitting: separando seus módulos para uma performance incrível'
```

Comment:
Muito "marketeiro"? Se for, podemos pensar em um título melhor. :)

Reply:
@wmartins Parabéns pelo artigo, ficou muito bom. Gostei do título, aproveitei e publiquei o texto. Obrigado!

Reply:
Opa, muito obrigado!